### ISSUE TITLE
Not loading cert-manager orders if there are many

### ISSUE THREAD

**Issue by biqas** [lensapp/lens#2332]
If there are many certificates created in the cluster about 10K then the certificate, orders and certifcaterequest listing is not working any more. I guess it is beacuse the amount is to big anthe lens try to read for each entry also the owned references which is leading to the problem.

**To Reproduce**
Create many certificates (10K).

**Comment by jim-docker** [comment]
How is it not working exactly? Does the spinner never go away? Are there any error messages?

**Comment by biqas** [comment]
No, there are no error messages, only loading spinner which does not go away. I guess it is trying to load all the time related data.
